src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -28,6 +28,8 @@
 @Service
 public class AttachCaseCallbackService {
 
+    public static final String INTERNAL_ERROR_MSG = "An error occurred";
```

Comment:
Because this message will be shown to the caseworker, maybe we should make it more descriptive, to make it clear it's the server error and maybe that it happened when trying to assign the exception record to the case.